packages/common/src/utils/validationUtils.ts

```diff
@@ -57,6 +57,10 @@ export const parseExpression = (
       }
       case CertificateDataValueType.BOOLEAN: {
         const valueBoolean = element.value as ValueBoolean
+
+        if ((valueBoolean.selected === undefined || valueBoolean.selected === null) && adjustedExpression.includes('not')) {
```

Comment:
How come we check "not"?

Reply:
Given we have two expressions, one that checks if a statement is true one that checks if its false, eg ($dodsdatumSakert & !$dodsdatumSakert).

This gets translated to "dodsdatumSakert" and "not dodsdatumSakert" in getExpression(). 

The first expression gets triggered if the statement is true, and the second if its false. Since the second statement (!$dodsdatumSakert) gets triggered if the statement is false, I added logic that returns true to not trigger negative expressions that are null/undefined.